recipes/sire/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "sire" %}
+{% set version = "2021.1.0" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/michellab/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: c029f78485dc52e060902c67d660fee35a3d655215bc9afb16ba17ca1b2a2627
+  patches: conda_build.patch
+
+build:
+  number: {{ build }}
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - git
+    - make
+  host:
+    - boost
+    - gsl
+    - libcblas
+    - libcpuid
+    - libnetcdf
+    - openmm
+    - python
+    - qt
+    - tbb-devel <=2020.2.0
+  run:
+    - boost
+    - gsl
+    - libcblas
+    - libcpuid
+    - libnetcdf
+    - openmm
+    - pymbar
+    - pyqt
+    - python
+    - qt
+    - tbb <=2020.2.0
```

Comment:
```suggestion
```

Reply:
Is there are reason why we don't need `tbb` as a run requirement? An `ldd` of our libraries would suggest otherwise.

Reply:
I assume that this is because of the [run_exports](https://github.com/conda-forge/tbb-feedstock/blob/master/recipe/meta.yaml#L76) section of the `tbb` recipe. As a reference, other packages such as  [deal.ii](https://github.com/conda-forge/dealii-feedstock/blob/master/recipe/meta.yaml) and [foamalgo](https://github.com/conda-forge/foamalgo-feedstock/blob/master/recipe/meta.yaml) list things explicitly, as I had done previously.

Is there a preference for not including a run requirement if it has an export? (I would have thought that being explicit was useful, so that someone could read your recipe without needing to cross-reference other dependencies to see what is pulled in. Obviously this gets complicated with dependencies of dependencies, and so on.)